documentation/modules/cruise-control/con-cruise-control-description.adoc

```diff
@@ -0,0 +1,25 @@
+//standard description for cruise control
+[role="_abstract"]
+Cruise Control is an open source system that supports the following Kafka operations:
+
+* Monitoring cluster workload
+* Rebalancing a cluster based on predefined constraints
+* Detecting and fixing anomalies.
```

Comment:
I know you are just describing what Cruise Control does here but I am worried this could set expectations to the users? Because "Detecting and fixing anomalies." is actually not supported in Strimzi.

Reply:
let's remove that bullet